### DISCUSSION TITLE
"Key Bindings" tab missing from "Config" panel??

### DISCUSSION THREAD

**Discussion by davidmcnabnz** [monocasual/giada#625] [General]
Hi all,

I'm getting started with Giada with a view to migrating from SooperLooper.

The Giada manual says the Config panel should have a Key bindings tab.

But when I open the Config panel on my own Giada instance, the Key Bindings panel does not show.

I'm using version 0.20.0+ds1-1, installed from Ubuntu 22.04 LTS apt feed.

Has the key bindings feature been intentionally removed? Or can it be enabled with a config file setting? Or do I need to fork the code and try to reinstate the panel from an earlier code revision?

Thanks in advance for any suggestions.
David

**Comment by gvnnz** [comment]
Hi @davidmcnabnz, we added the key binding panel in [Giada v0.21](https://github.com/monocasual/giada/releases/tag/v0.21.0). You could download the AppImage package we provide [here](https://giadamusic.com/download) to get the latest release available. Cheers

**Reply by davidmcnabnz** [reply]
@gvnnz thanks for this. I look forward to trying it. :)
